lib/stripe/stripe_client.rb

```diff
@@ -53,30 +54,45 @@ def self.active_client
     # clears them from internal tracking in all connection managers across all
     # threads.
     #
+    # If passed a `config` object, only clear connection managers for that
+    # particular configuration.
+    #
     # For internal use only. Does not provide a stable API and may be broken
     # with future non-major changes.
-    def self.clear_all_connection_managers
+    def self.clear_all_connection_managers(config: nil)
```

Comment:
the "all" in the name is now slightly weird

Reply:
Yeahhhh, I thought about that too. Just given that this is Ruby and everything is allowed to call everything, I opted just to leave it.

The name is still somewhat apt because you can have multiple connection managers for a single config, so you can read this as "clear all the connection managers for config X".